lib/node-utils.ts

```diff
@@ -181,3 +181,48 @@ export function isRenderVariableDeclarator(
 
   return false;
 }
+
+type GetTestingLibraryUtilImportArgs = {
+  context: any,
+  utilName: string,
+  extraModules?: string[],
+};
+
+export function getTestingLibraryUtilImport(options: GetTestingLibraryUtilImportArgs): null | string {
```

Comment:
This is the method to find any testing library import from `@testing-library/framework` or custom module defined in ESLint shared settings. It just needs 3 args:

- context: from the rule, so from there it can get the source code with `context.getSourceCode()` to find all imports in different formats, and to get `context.settings` with plugin options.
- utilName: this is the actual Testing Library util you are looking for (e.g. "render", "screen", "waitFor")
- extraModules: an optional one to cover user-event, as it can be imported from custom module or its own package as it doesn't belong to Testing Library frameworks. It could be useful in other cases too.